repos/fdbt-site/src/pages/products/productDetails.tsx

```diff
@@ -64,18 +65,49 @@ const ProductDetails = ({
     </TwoThirdsLayout>
 );
 
+const getReadValue = (element: ProductDetailsElement) => {
+    return element.content.map((item) => (
+        <span key={item} id={element.id || undefined}>
+            {item}
+        </span>
+    ));
+};
+
+const getEditableValue = (element: ProductDetailsElement) => {
+    return (
+        <div className="dft-flex dft-flex-justify-space-between">
+            {element.content.map((item) => {
+                return (
+                    <span key={item} id={element.id || undefined}>
+                        {item}
+                    </span>
+                );
+            })}
+
+            <a href={element.editLink}>Edit</a>
+        </div>
+    );
+};
+
 export const getServerSideProps = async (ctx: NextPageContextWithSession): Promise<{ props: ProductDetailsProps }> => {
     const noc = getAndValidateNoc(ctx);
-    const productId = ctx.query?.productId;
+
     const serviceId = ctx.query?.serviceId;
 
+    const productId = ctx.query?.productId;
```

Comment:
Could we add the productId check in the if statement below? Would make sense to check them both at the same time, as we should fail immediately if they arent there.

Reply:
I think also we only need to check if the type isnt a string, because undefined and string[] are both incorrect.